packages/react-components/playwright/index.tsx

```diff
@@ -27,6 +26,7 @@ export type HooksConfig = {
 beforeMount<HooksConfig>(async ({ App, hooksConfig }) => {
   // If useTheme is not provided, default to true
   const useTheme = hooksConfig?.useTheme === undefined ? true : hooksConfig?.useTheme;
+  initializeFileTypeIcons();
```

Comment:
initialize icons functions should be used outside of functions and should be called only once for an app

Reply:
Please see https://github.com/microsoft/fluentui/wiki/Using-icons#font-based-icons for more details

Reply:
thanks updated